lib/nexthop_group.c

```diff
@@ -482,9 +489,10 @@ static void nexthop_group_write_nexthop_internal(struct vty *vty,
 {
 	char buf[100];
 
-	vty_out(vty, "nexthop ");
+	vty_out(vty, "nexthop");
```

Comment:
Shouldn't this be part of the last commit?

Reply:
This change belongs to the first commit because both `nh->addr` and `nh->intf` are optional now, so we need to display the nexthop like this:
```
        vty_out(vty, "nexthop");

        if (nh->addr)
                vty_out(vty, " %s", ...);

        if (nh->intf)
                vty_out(vty, " %s", ...);
```

The previus code assumed `nh->addr` was always present.